api/src/controllers/events/click-event.ts

```diff
@@ -29,4 +29,22 @@ const clickEventRoute = APIWrapper({
     },
 });
 
-export const clickEvent = relogRequestHandler(clickEventRoute);
\ No newline at end of file
+const clickEventPaginationRoute = APIWrapper({
+    GET: {
```

Comment:
Use unified object for routing - see: https://github.com/GTBitsOfGood/bog-analytics/pull/21/files#r1493344769